src/topology/bounded_continuous_function.lean

```diff
@@ -466,7 +466,10 @@ sub_le_iff_le_add'.1 $ (abs_le.1 $ @dist_coe_le_dist _ _ _ _ f g x).2
 end normed_group
 
 section normed_space
-/-! In this section, if `β` is a normed space, then we show that the space of bounded
+/-!
```

Comment:
Could you use the same kind of markup comments in the sections below?

Reply:
Modified in 29e139f, I hope I have picked up the correct style.